packages/data-stores/src/woocommerce-eligibility/constants.ts

```diff
@@ -0,0 +1,41 @@
+export const STORE_KEY = 'automattic/woocommerce-eligibility';
+
+export const eligibilityHolds = {
+	BLOCKED_ATOMIC_TRANSFER: 'BLOCKED_ATOMIC_TRANSFER',
+	TRANSFER_ALREADY_EXISTS: 'TRANSFER_ALREADY_EXISTS',
+	NO_BUSINESS_PLAN: 'NO_BUSINESS_PLAN',
+	NO_JETPACK_SITES: 'NO_JETPACK_SITES',
+	NO_VIP_SITES: 'NO_VIP_SITES',
+	SITE_PRIVATE: 'SITE_PRIVATE',
+	// SITE_UNLAUNCHED is a client constant to differentiate between launched private sites, and unlaunched sites.
+	// See: client/state/data-layer/wpcom/sites/automated-transfer/eligibility/index.js
+	SITE_UNLAUNCHED: 'SITE_UNLAUNCHED',
+	SITE_GRAYLISTED: 'SITE_GRAYLISTED',
+	NON_ADMIN_USER: 'NON_ADMIN_USER',
+	NOT_RESOLVING_TO_WPCOM: 'NOT_RESOLVING_TO_WPCOM',
+	NO_SSL_CERTIFICATE: 'NO_SSL_CERTIFICATE',
+	EMAIL_UNVERIFIED: 'EMAIL_UNVERIFIED',
+	EXCESSIVE_DISK_SPACE: 'EXCESSIVE_DISK_SPACE',
+};
+
+/**
+ * Maps the constants used in the WordPress.com API with
+ * those used inside of Calypso. Somewhat redundant, this
+ * provides safety for when the API changes. We need not
+ * changes the constants in the Calypso side, only here
+ * in the code directly dealing with the API.
+ */
+export const statusMapping = {
+	blocked_atomic_transfer: eligibilityHolds.BLOCKED_ATOMIC_TRANSFER,
```

Comment:
Seems like there's not a good reason to hold the eligibility reasons in a `eligibilityHolds` object. They can be constants, accessible only to the module, or even better - inlined if there's no other use for them when declared separately as constants.

Reply:
This was code ported over from elsewhere but I can try and clean it up a bit.